pkg/monitors/telegraf/monitors/dns/metadata.yaml

```diff
@@ -0,0 +1,33 @@
+monitors:
+- doc: |
+    This is an embedded form of the [Telegraf DNS Query
+    plugin](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/dns_query).
+
+  dimensions:
+    plugin:
+      description: The plugin name "telegraf/dns".
+    domain:
+      description: The DNS domain queried.
+    server:
+      description: The DNS server queried.
+    record_type:
+      description: The DNS record type queried.
+    result:
+      description: Resolution result string (success = 0, timeout = 1, error = 2).
```

Comment:
This should be a metric value, not a dimension.

Reply:
done